index.html

```diff
@@ -50,7 +50,8 @@
 </head>
 
 <body>
-  <section id='abstract' data-include="spec/01-abstract.md" data-include-format='markdown'></section>
+  <section id='abstract' data-include="spec/00-abstract.md" data-include-format='markdown'></section>
+  <section id='abstract' data-include="spec/01-processing-model.md" data-include-format='markdown'></section>  
```

Comment:
remove `id='abstract'` from second one

```suggestion
  <section data-include="spec/01-processing-model.md" data-include-format='markdown'></section>  
```

Reply:
Only one should have this `id`